rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec

```diff
@@ -1,6 +1,8 @@
 %global gem_name foreman_scap_client
 %global config_dir %{_sysconfdir}/%{gem_name}
 
+%define rubyabi 1.9.1
```

Comment:
If you're not going to add the definition, then maybe remove the usage of `%{rubyabi}` instead?